code/modules/reagents/Chemistry-Recipes_vr.dm

```diff
@@ -205,7 +205,14 @@
 				O.show_message(text("<span class='warning'>The slime core fizzles disappointingly.</span>"), 1)
 			return
 
-		var/blocked = list(/obj/item/stack/material, /obj/item/stack/material/cyborg, /obj/item/stack/material/cyborg/plastic, /obj/item/stack/material/cyborg/plasteel, /obj/item/stack/material/cyborg/glass/reinforced, /obj/item/stack/material/cyborg/wood, /obj/item/stack/material/animalhide/human, /obj/item/stack/material/animalhide/corgi, /obj/item/stack/material/animalhide/cat, /obj/item/stack/material/animalhide/monkey, /obj/item/stack/material/animalhide/lizard , /obj/item/stack/material/animalhide/xeno, /obj/item/stack/material/cyborg, /obj/item/stack/material/cyborg/glass/reinforced)
+		var/blocked = list(
+							/obj/item/stack/material,					//Technical stacks
+							/obj/item/stack/material/animalhide,
```

Comment:
This should probably be a typesof, since normally most animal hides are blocked anyway.

Reply:
They were blocked but change to not make them blocked other than base technical type was itnentional, unless there was some specific purpose to the animalhides not being available?

Reply:
I am guessing it was not intended for animal hides to be produced via the slime reaction. Why? I cannot tell you, but probably should keep it that way for the time being, since adding a bunch of animal hides also substantially effects balance.